students/Lincoln_Zhang/Session7/test_html_output2.html

```diff
@@ -0,0 +1,2 @@
+None<html_render.Body object at 0x7f0938b8d160>
```

Comment:
It looks like you are not printing out the contents of the Element, instead it is printing out the str version of the object.

Reply:
I can give some partial credit here since it looks like you have some logic but you are missing some implementation in the render